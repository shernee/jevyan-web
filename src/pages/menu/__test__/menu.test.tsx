import React from 'react'
import { shallow } from 'enzyme'
import Menu from '../index'

describe('Menu page', () => {
  it('should render correctly', () => {
    const component = shallow(<Menu />)
    expect(component).toMatchSnapshot()
  })
})
