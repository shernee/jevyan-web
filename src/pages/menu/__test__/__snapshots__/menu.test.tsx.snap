// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="menu-page"
>
  <DeliveryPickModal
    dayList={
      Array [
        "Thursday, Oct 29th",
        "Friday, Oct 30th",
        "Saturday, Oct 31st",
      ]
    }
    handleModalChange={[Function]}
    show={true}
    timeList={
      Array [
        "9:00 AM-1:00 PM",
        "2:00 PM-6:00 PM",
      ]
    }
  />
  <div
    className="dropdown-header"
  >
    <WithStyles(ForwardRef(NativeSelect))
      inputProps={
        Object {
          "style": Object {
            "textAlign": "center",
          },
        }
      }
      name="day"
      onChange={[Function]}
      value=""
    >
      <option
        disabled={true}
        value=""
      />
      <option
        key="0"
        value="Thursday, Oct 29th"
      >
        Thursday, Oct 29th
      </option>
      <option
        key="1"
        value="Friday, Oct 30th"
      >
        Friday, Oct 30th
      </option>
      <option
        key="2"
        value="Saturday, Oct 31st"
      >
        Saturday, Oct 31st
      </option>
    </WithStyles(ForwardRef(NativeSelect))>
    <WithStyles(ForwardRef(NativeSelect))
      inputProps={
        Object {
          "style": Object {
            "textAlign": "center",
          },
        }
      }
      name="time"
      onChange={[Function]}
      value=""
    >
      <option
        disabled={true}
        value=""
      />
      <option
        key="0"
        value="9:00 AM-1:00 PM"
      >
        9:00 AM-1:00 PM
      </option>
      <option
        key="1"
        value="2:00 PM-6:00 PM"
      >
        2:00 PM-6:00 PM
      </option>
    </WithStyles(ForwardRef(NativeSelect))>
  </div>
  <Banner
    bannerData={
      Object {
        "city": "",
        "currency": "",
        "name": "",
        "phone": "",
        "postal": "",
        "state": "",
        "street": "",
      }
    }
  />
  <div
    className="below-banner-section"
  >
    <CategoryHeader
      cartQuantity={0}
      categories={Array []}
    />
    <MenuCard
      categories={Array []}
      currency=""
      items={Array []}
    />
  </div>
</div>
`;

exports[`Menu page should render correctly 1`] = `
<div
  className="menu-page"
>
  <DeliveryPickModal
    dayList={
      Array [
        "Thursday, Oct 29th",
        "Friday, Oct 30th",
        "Saturday, Oct 31st",
      ]
    }
    handleModalChange={[Function]}
    show={true}
    timeList={
      Array [
        "9:00 AM-1:00 PM",
        "2:00 PM-6:00 PM",
      ]
    }
  />
  <div
    className="dropdown-header"
  >
    <WithStyles(ForwardRef(NativeSelect))
      inputProps={
        Object {
          "style": Object {
            "textAlign": "center",
          },
        }
      }
      name="day"
      onChange={[Function]}
      value=""
    >
      <option
        disabled={true}
        value=""
      />
      <option
        key="0"
        value="Thursday, Oct 29th"
      >
        Thursday, Oct 29th
      </option>
      <option
        key="1"
        value="Friday, Oct 30th"
      >
        Friday, Oct 30th
      </option>
      <option
        key="2"
        value="Saturday, Oct 31st"
      >
        Saturday, Oct 31st
      </option>
    </WithStyles(ForwardRef(NativeSelect))>
    <WithStyles(ForwardRef(NativeSelect))
      inputProps={
        Object {
          "style": Object {
            "textAlign": "center",
          },
        }
      }
      name="time"
      onChange={[Function]}
      value=""
    >
      <option
        disabled={true}
        value=""
      />
      <option
        key="0"
        value="9:00 AM-1:00 PM"
      >
        9:00 AM-1:00 PM
      </option>
      <option
        key="1"
        value="2:00 PM-6:00 PM"
      >
        2:00 PM-6:00 PM
      </option>
    </WithStyles(ForwardRef(NativeSelect))>
  </div>
  <Banner
    bannerData={
      Object {
        "city": "",
        "currency": "",
        "name": "",
        "phone": "",
        "postal": "",
        "state": "",
        "street": "",
      }
    }
  />
  <div
    className="below-banner-section"
  >
    <CategoryHeader
      cartQuantity={0}
      categories={Array []}
    />
    <MenuCard
      categories={Array []}
      currency=""
      items={Array []}
    />
  </div>
</div>
`;
